src/app/TopicDasboard/ResourceEditForm.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//TODO:remove console.logs
+export function ResourceEditForm({ resource, handleUpdate }) {
+  return (
+    <div>
+      <form
+        resourceid={resource.id}
+        className="resource-item"
+        onSubmit={e => {
```

Comment:
For performance reasons, it might be a lot better to make this a class component and factor this out as a method, instead of inlining your onSubmit function here.